pages/common/pydocstyle.md

```diff
@@ -0,0 +1,32 @@
+# pydocstyle
+
+> A static analysis tool for checking compliance with Python docstring conventions.
```

Comment:
```suggestion
> Statically check Python scripts for compliance with docstring conventions.
```
We start with a verb in the imperative form always when possible.

Reply:
Done.